pkg/policy/mapstate_test.go

```diff
@@ -1426,6 +1426,42 @@ func (ds *PolicyTestSuite) TestMapState_AccumulateMapChangesDeny(c *check.C) {
 			DNSUDPEgressKey(41):     {},
 		},
 		deletes: Keys{},
+	}, {
+		continued: false,
+		name:      "test-7a - egressDeny ports and toFQDN",
+		setup: newMapState(map[Key]MapStateEntry{
+			// egressDeny toPorts TCP/80
+			testEgressKey(0, 80, 6): denyEntry(0, csFoo),
+		}),
+		args: []args{
+			// Simulate addition of toFQDN identity
+			{cs: csFoo, adds: []int{16777218}, deletes: []int{}, port: 80, proto: 6, ingress: false, redirect: false, deny: true},
+			{cs: csBar, adds: []int{16777218}, deletes: []int{}, port: 0, proto: 0, ingress: false, redirect: false, deny: false},
```

Comment:
Does it make a difference / help coverage if we tested with a different identity as well?

Reply:
not sure what you mean - I've chosen this identity for being locally scoped because that was the case I had in mind, but I'm not sure it matters.
By different identity you mean an additional identity or a different numeric identity?

Reply:
Sorry, I meant a different numeric identity.

Reply:
I don't believe so, the mapstate seems mostly angostic to whether an ID is locally scoped or not.